pkg/ccl/backupccl/backupdestination/backup_destination.go

```diff
@@ -6,68 +6,70 @@
 //
 //     https://github.com/cockroachdb/cockroach/blob/master/licenses/CCL.txt
 
-package backupccl
+package backupdestination
```

Comment:
I don't know how others field, but I could live with `backupdest`.

Also, it might be nice to have some package level documentation comment about what our intent is for this package.

Reply:
I'm +1 on backupdest, any objections @dt @benbardin?

Reply:
Changed to `backupdest` for the time being.